cayleypy/graphs_lib.py

```diff
@@ -92,6 +97,37 @@ def prepare_graph(name, n=0) -> CayleyGraph:
             generators.append(perm)
             generator_names.append("R" + str(prefix_len - 1))
         return CayleyGraph(generators, dest=list(range(n)), generator_names=generator_names)
+    elif name == "cubic_pancake":
+        assert n >= 2
+        subset = kwargs['S']
+        assert subset in [1, 2, 3, 4, 5, 6, 7]
+
+        def PuncakeGenerator(k, n):
```

Comment:
Name it "pancake_generator", add type annotations.
https://google.github.io/styleguide/pyguide.html#316-naming